src/Components/EditToggleButton.tsx

```diff
@@ -11,15 +12,27 @@ interface EditToggleButtonProps {
     onClick?: (newState: boolean, oldState: boolean)=>void;
 }
 
+const selectedButton = (
```

Comment:
I go back and forth on inlining vs not inlining... It's nice to be inlined for readability, but it's better to be up top for reuse